community/views.py

```diff
@@ -1,28 +1,109 @@
 from django.http import HttpResponse
 from django.views.generic.base import TemplateView
-
+import requests
 from trav import Travis
 
 from .git import (
     get_deploy_url,
     get_org_name,
     get_owner,
     get_upstream_deploy_url,
+    get_remote_url
 )
+from data.models import Team
+from gamification.models import Participant as GamificationParticipant
+from meta_review.models import Participant as MetaReviewer
+
+
+def initialize_org_context_details():
+    org_name = get_org_name()
+    org_details = {
+        'name': org_name,
+        'blog_url': 'https://blog.{}.io/'.format(org_name),
+        'twitter_url': 'https://twitter.com/{}_io/'.format(org_name),
+        'facebook_url': 'https://www.facebook.com/{}Analyzer'.format(
+            org_name),
+        'repo_url': get_remote_url().href,
+        'docs': 'https://{}.io/docs'.format(org_name),
+        'newcomer_docs': 'https://{}.io/newcomer'.format(org_name),
+        'coc': 'https://{}.io/coc'.format(org_name),
+        'logo_url': ('https://api.{}.io/en/latest/_static/images/'
+                     '{}_logo.svg'.format(org_name, org_name)),
+        'gitter_chat': 'https://gitter.im/{}/{}/'.format(org_name,
+                                                         org_name),
+        'github_core_repo': 'https://github.com/{}/{}/'.format(org_name,
+                                                               org_name),
+        'licence_type': 'GNU AGPL v3.0'
+    }
+    return org_details
+
+
+def get_header_and_footer(context):
+    context['isTravis'] = Travis.TRAVIS
+    context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    context['org'] = initialize_org_context_details()
+    print('Running on Travis: {}, build link: {}'.format(context['isTravis'],
+                                                         context['travisLink']
+                                                         ))
+    return context
 
 
 class HomePageView(TemplateView):
     template_name = 'index.html'
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['isTravis'] = Travis.TRAVIS
-        context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    def get_team_details(self, org_name):
+        teams = [
+            '{} newcomers'.format(org_name),
+            '{} developers'.format(org_name),
+            '{} admins'.format(org_name)
+        ]
+        team_details = {}
+        for team in teams:
+            team = Team.objects.get(name=team)
+            contributors = team.contributors.filter(teams=team)
+            team_details[team] = contributors.count()
+        return team_details
+
+    def get_quote_of_the_day(self):
+        qod = requests.get('http://quotes.rest/qod?category=inspire')
+        qod_data = qod.json()
+        if qod.status_code == 429:
```

Comment:
Can you do something like https://github.com/coala/community/blob/9a7b5f4a32831771ea3e5d1b9befd8e7311b886d/data/contrib_data.py#L14